partner_compassion/models/partner_compassion.py

```diff
@@ -389,6 +462,33 @@ def action_view_partner_invoices(self):
         action["context"] = {"search_default_partner_id": self.id}
         return action
 
+    @api.multi
+    def notify_event_responsible_criminal_record_expiration(self):
+        """
+        Action rule called when a criminal record expires. A notification is sent
+        to the event responsible.
+        """
+        # search for event responsible(s) of all events attended by partner
+        for partner in self:
+            registrations = self.env['event.registration'].sudo().search([
+                ("partner_id", "=", partner.id)
+            ])
+            for registration in registrations:
+                # get responsible
+                event = registration.event_id
+                responsible = event.user_id
```

Comment:
You shouldn't take the responsible of the event. Some people may also have a criminal record but not any event registrations, so this should work regardless of events. You should add a setting to define the user to notify inside the Compassion Settings (by inheriting res.config.settings like it's done in other modules) and use this setting to select the user.

Reply:
Corrected in commit 643bc23